src/common/translate/lang/en.json

```diff
@@ -164,9 +164,11 @@
     "NETWORK_PATH": "Network (Derivation path)",
     "REVIEW_SECURITY_DETAILS": "Review Security Details",
     "NONCE_CHANGED": "Transaction nonce has been changed",
+    "NONCE_CHANGED_CONTENT": "This transaction had the same nonce as a previously sent transaction. We've changed it for you, but you may edit it if you need to, using the pencil icon below.",
     "NONCE_CONFLICT_IN_QUEUE": "Nonce is used in another transaction",
-    "NONCE_CHANGED_HELP": "This transaction had the same nonce as a previously sent Transaction. We’ve changed it for you but you may edit it if you need to using the pencil icon above.",
+    "NONCE_CHANGED_HELP": "This transaction had the same nonce as a previously sent transaction. We’ve changed it for you, but you may edit it if you need to using the pencil icon above.",
     "NONCE_OUT_OF_ORDER": "Approving TX out of order affects nonce",
+    "NONCE_OUT_OF_ORDER_CONTENT": "Nonces have be pre-assigned by app.mycrypto.com. You may approve/deny the transactions out of order, but all transactions initiated after this one will be automatically denied.",
```

Comment:
This should probably be dynamic too right?

Reply:
We don't have a good way to pass stuff to the banners, with how it's currently written. That will require a bit of a refactor probably.

Reply:
Okay, just make sure to make a card for it then.